crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -20,9 +20,21 @@ module Installer
   module UpgradesHelper
     def upgrade_continue_button(disabled = false)
       if disabled
-        button_tag icon_tag(:chevron_right, t(".continue")), class: "btn btn-primary disabled"
+        button_tag(
+          icon_tag(:chevron_right, t(".continue")),
+          class: "btn btn-primary disabled",
+          data: {
+            blockui: t(".blockui")
```

Comment:
The helper uses the `blockui` text from the current page namespace, so we can't set it explicit.